MBBSEmu/Configuration.cs

```diff
@@ -64,6 +64,17 @@ public AppSettings(ILogger logger)
         public string TelnetIPAddress => GetIPAppSettings("Telnet.IP");
         public string RloginIPAddress => GetIPAppSettings("Rlogin.IP");
         
+        public Session.Rlogin.EnumRloginCompatibility RloginCompatibility
+        {
+            get
+            {
+                if (!ConfigurationRoot.GetSection("Rlogin.Compatibility").Exists())
+                    return Session.Rlogin.EnumRloginCompatibility.Default;
+                    
+                return (Session.Rlogin.EnumRloginCompatibility)Enum.Parse(typeof(Session.Rlogin.EnumRloginCompatibility), ConfigurationRoot["Rlogin.Compatibility"]);
```

Comment:
Change this to an `Enum.TryParse` to be safer if they type in a value that can't be matched

Reply:
It is not a drop in replacement, so I changed things around a bit.